src/main/target/HAKRCF405/target.h

```diff
@@ -0,0 +1,172 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "HKF4"
+#define USBD_PRODUCT_STRING  "HAKRCF405"
+//----------------------------------------
+
+//LED & BEE------------------------------- 
+#define LED0_PIN                PC13
```

Comment:
Please swap this pin with the pin for `BEEPER` right below. This way you can use the DALRCF405 target that this is a copy of.

Reply:
We haved redesign must GPIO defined, and is diffrent of DALRCF405 . And add use new baro sensor device.

Reply:
Just swap it back and use the DALRCF405 target that you are cloning.

Reply:
Only the code style  is  as same as the DALRCF405.

Reply:
No, the target is the same, just with some copy paste done (and a bug added):